### PR TITLE
Update key packages

### PR DESCRIPTION
**Pull request by elevu**
In this PR several key packages have been updated including: nuxt, bootstrap, lint

**How to prepare for test**:
- [x] ssh to clinical-db
- [x] install on stage of clinical-db:
`bash servers/resources/clinical-db.sclifelab.se/update-cgweb-stage.sh [THIS-BRANCH-NAME]`

**How to test**:
- [x] login on https://clinical-stage.scilifelab.se/
- [x] test all the app functionalities, tables and form submission

**Expected test outcome**:
- [x] check that everything works as expected.

**Review:**
- [x] code approved by PG
- [x] tests executed by @elevu 
- [x] "Merge and deploy" approved by PG
Thanks for filling in who performed the code review and the test!

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by patrikgrenfeldt** [comment]
I get: 
image
But it does not seem to fail the build, I think this is a new behavior. The old build process always failed on all warnings and I think it is quite a good way of keeping the warnings at 0. Could you make it fail again on warnings?

**Comment by patrikgrenfeldt** [comment]
Test node lts-10 node:
image
image
👍

**Comment by elevu** [comment]
@patrikgrenfeldt  Updated the invalid form input look to the new Bootstrap syntax: 
Screen Shot 2020-08-25 at 13 03 21

**Comment by elevu** [comment]
Screen Shot 2020-08-26 at 08 46 09
Screen Shot 2020-08-26 at 08 45 39
@patrikgrenfeldt Updated the texts, only some of them are showing in the screenshot but all of them were changed

File: package.json [hunk 1]
```diff
@@ -18,33 +18,33 @@
...
-    "eslint-loader": "1.x.x",
-    "eslint-plugin-html": "3.x.x",
+    "eslint-loader": "4.x.x",
+    "eslint-plugin-html": "6.x.x",
     "eslint-plugin-import": "2.x.x",
     "eslint-plugin-node": "5.x.x",
     "eslint-plugin-promise": "3.x.x",
-    "eslint-plugin-standard": "3.x.x"
+    "eslint-plugin-standard": "3.x.x",
+    "eslint-plugin-vue": "^7.0.0-beta.2"
```
[the remark this answers is not in the document]
Comment (patrikgrenfeldt) [reply package.json]: discussed face to face

File: package.json [hunk 2]
```diff
@@ -18,33 +18,33 @@
...
     "vue-chartjs": "3.x.x",
     "vue-google-auth": "1.x.x",
     "vue-multiselect": "2.x.x",
     "webpack": "4.x.x"
   },
   "devDependencies": {
-    "babel-eslint": "8.x.x",
-    "eslint": "4.x.x",
+    "babel-eslint": "10.x.x",
+    "eslint": "^7.7.0",
```
[the remark this answers is not in the document]
Comment (patrikgrenfeldt) [reply package.json]: discussed face to face

**Pull request merged**